tests/src/onhost_tests/calibration_handler_test.cpp

```
@@ -7,8 +8,8 @@
 
 using namespace dai;
 
-static ImuModelParams makeImuModelParams() {
-    ImuModelParams params;
+static ImuNoiseParameters makeimuNoiseParams() {
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)